src/components/checkout/payment-methods.js

```diff
@@ -0,0 +1,53 @@
+// Libraries
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+
+// Lib
+import componentMapping from '../../lib/component-mapping'
+
+class PaymentMethods extends Component {
+  constructor (props) {
+    super(props)
+    this.Button = componentMapping('Button')
+    this.PaymentMethodHeader = componentMapping('PaymentMethodHeader')
+    this.PayPalButton = componentMapping('PayPalButton')
+  }
+
+  /**
+   * When a customer selects the default checkout option, this funtion is called and 
+   * it sets the selected payment method in the state and redirects to the next section
+   */
+  handleDefaultPaymentSelection(paymentMethod) {
+    const { nextSection, handleSetPaymentMethod } = this.props
+    handleSetPaymentMethod(paymentMethod)
+    nextSection()
```

Comment:
I presume `handleSetPaymentMethod` will trigger a Redux store update which is asynchronous. I am wondering whether `nextSection` should wait until `handlerSetPaymentMethod` completes?

This could be done by chaining promises or by handling navigation within `handleSetPaymentMethod`.